DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs

```diff
@@ -65,7 +65,7 @@ public void Section_content_should_have_consolidation_path_when_courseSetting_ex
                 consolidationPath: consolidationPath,
                 courseSettings: null
             );
-            var expectedConsolidationPath = ContentUrlHelper.GetContentPath(config, consolidationPath);
+            var expectedConsolidationPath = config.GetConsolidationPathUrl(consolidationPath);
```

Comment:
Please put the actual path in here, instead of relying on the implementation of the config extension method. Otherwise we're not actually testing that the value is correct 🙂

Reply:
Should I still use config["CurrentSystemBaseUrl"] in the test?

Reply:
My usual process is:
* Store the mock base url in a variable 
* Set config to return that base url as the CurrentSystemBaseUrl
* Check that the result is the mock base url + the path you expect
Does that make sense? For example see `Content_viewer_should_parse_scorm_source` in `ContentViewerViewModelTests`  🙂